libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.manager.spec.ts

```diff
@@ -0,0 +1,204 @@
+import { Kysely } from 'kysely';
+
+import { DB, testAccountDatabaseSetup } from '@fxa/shared/db/mysql/account';
+
+import { CreatePaypalCustomerFactory } from './paypalCustomer.factories';
+import { PaypalCustomerManager } from './paypalCustomer.manager';
+import {
+  PaypalCustomerNotCreatedError,
+  PaypalCustomerNotDeletedError,
+  PaypalCustomerNotFoundError,
+  PaypalCustomerNotUpdatedError,
+} from './paypalCustomer.error';
+
+describe('PaypalManager', () => {
+  let kyselyDb: Kysely<DB>;
+  let paypalCustomerManager: PaypalCustomerManager;
+
+  beforeAll(async () => {
+    kyselyDb = await testAccountDatabaseSetup(['paypalCustomers']);
+
+    paypalCustomerManager = new PaypalCustomerManager(kyselyDb);
+  });
+
+  afterAll(async () => {
+    if (kyselyDb) {
+      await kyselyDb.destroy();
+    }
+  });
+
+  describe('createPaypalCustomer', () => {
+    it('creates a paypalCustomer successfully', async () => {
+      const paypalCustomer = CreatePaypalCustomerFactory();
+
+      const result = await paypalCustomerManager.createPaypalCustomer(
+        paypalCustomer
+      );
+
+      expect(result).toEqual({
+        ...paypalCustomer,
+        createdAt: expect.anything(),
+      });
+    });
+
+    it('throws a PaypalCustomerNotCreatedError when the DB throws', async () => {
+      const paypalCustomer = CreatePaypalCustomerFactory();
+
+      paypalCustomer.billingAgreementId =
```

Comment:
nit. Could this be passed in as an override param for CreatePaypalCustomerFactory?

Reply:
Updated!